consensus/config.go

```diff
@@ -19,7 +19,7 @@ const (
 	proposingTimeout              = chain.ProposingTimeTolerance * 4 / 5
 	cacheExpiration               = 3600 * time.Second
 	cacheCleanupInterval          = 600 * time.Second
-	proposingStartDelay           = config.ConsensusTimeout + time.Second
+	proposingStartDelay           = 2*config.ConsensusTimeout + time.Second
```

Comment:
Why delay consensus timeout from 61s to 121s ?
It will cause mainnet daily yields declined.

Reply:
Because we now have another 60s to connect to random neighbors. This delay only applies once when nknd first starts